lib/tasks/scrape.rb

```diff
@@ -0,0 +1,9 @@
+#Pass in a single string with your SIS Username and Password, string delimited.
+require_relative './SISScraper.rb'
```

Comment:
Rails convention is to snake case the file name, so 'sis_scraper.rb', and put it in 'libs/'. This is in Rails path, so then in this file we can just do `require 'sis_scraper'`.